app/js/components/labOrderEntry/DraftOrder/DraftList.jsx

```diff
@@ -1,20 +1,59 @@
 import React from 'react';
 import shortid from 'shortid';
 import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExclamationTriangle, faCheckSquare } from '@fortawesome/free-solid-svg-icons';
+
 
 export class DraftList extends React.Component {
+  static toggleIcon(order) {
+    let orderUrgencyIcon;
+    if (!Object.prototype.hasOwnProperty.call(order, 'urgency') || order.urgency === 'routine') {
```

Comment:
if you did not need to manage state then why move the component out of the `LabEntryForm` cos now props is being passed down twice

Reply:
The component is still in `LabEntryForm` as `LabDraftOrder`.
I believe there are more reasons to create a component other than it not needing its own state. It makes it easier to debug, as your error stack traces its error to a very specific component.
Also, this would not cause a performance issue I believe.

I moved it up one layer tho, so `LabDraftOrder` should be the main component now.

Reply:
Fixed